dart/biomechanics/DynamicsFitter.cpp

```diff
@@ -11642,6 +11646,12 @@ std::pair<bool, double> DynamicsFitter::zeroLinearResidualsAndOptimizeAngular(
     {
       numMissing = maxBuckets;
     }
+    if (numMissing > 200)
+    {
+      std::cout << "More than 200 timesteps marked as missing GRF. Aborting..."
+                << std::endl;
+      return {false, totalResidual};
+    }
```

Comment:
Did you mean to include this on this PR? Just checking, since you didn't mention it in the comment.

Reply:
Oh, yes, I just snuck this in here cause I've been meaning to add an escape hatch and didn't want to open a separate PR. I hope this doesn't break anything...